### PR TITLE
NO_ISSUE: Adding the missing function.

### PR DESCRIPTION
**Pull request by achuzhoy**
Signed-off-by: Alexander Chuzhoy <achuzhoy@redhat.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Comment by achuzhoy** [comment]
cc @flaper87

File: deploy-lvmo/deploy.sh [hunk 1]
```diff
@@ -43,6 +43,12 @@ function extract_vars() {
     export CHANGEME_STORAGE_DEVICE_SET_COUNT="${disks_count}"
 }
 
+function extract_kubeconfig() {
```
Comment (flaper87) [inline deploy-lvmo/deploy.sh:46]: This one should come from `common.sh`

I'm assuming the execution failed for you, right? Let's try to see why it didn't get the function from `common.sh`

Comment (achuzhoy) [reply deploy-lvmo/deploy.sh:46]: Without it  the failure is:
[deploy-lvmo : deploy-lvmo] ./verify.sh: line 20: extract_kubeconfig: command not found

Comment (flaper87) [reply deploy-lvmo/deploy.sh:46]: then we should probably just call `extract_kubeconfig_common` instead. I think it does exactly what you need here :)

Comment (flaper87) [reply deploy-lvmo/deploy.sh]: Yeah, I started doing some of that a couple of PRs ago. Still lots to clean-up :)

**Review by flaper87 — APPROVED** [review]

**Review by eifrach — APPROVED** [review]

**Pull request merged**